### PR TITLE
docker: Upgrade to node v18

### PR DESCRIPTION
**Pull request by rye**

### CODE REVIEW

**Comment by hawkrives** [comment]
How about node 18?

**Comment by drewvolz** [comment]
Here's the error for the last build:
```sh
gyp ERR! stack Error: not found: make
```

We could probably go to node 18, is there a reason for us to use 16 over 18 here?

**Comment by hawkrives** [comment]
Nope

**Comment by rye** [comment]
@hawkrives @drewvolz This branch should now be ready to review. I squashed the commits so this is just a series of 3 commits — upgrading the actual version, fixing the Docker build, and then fixing the CI config.

File: .circleci/config.yml [hunk 1]
```diff
@@ -2,10 +2,10 @@
 version: 2
 
 x-config:
-  - &name "frogpond/ccc-server"
+  - &name 'frogpond/ccc-server'
   - &defaults
     docker:
-      - image: &default-image docker.io/circleci/node:10
+      - image: &default-image docker.io/circleci/node:12
```
Comment (drewvolz) [inline .circleci/config.yml:8]: Should this be 18, too?

**Review by drewvolz — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (rye) [reply .circleci/config.yml:8]: No such image exists — I think this is a debian version or something?

**Pull request merged**